app/views/application/_usa_banner.html.erb

```diff
@@ -24,6 +24,9 @@
       </div>
     </header>
     <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+      <script>
+        document.getElementById('gov-banner').setAttribute('hidden', '');
```

Comment:
This prevents the banner flash from occurring.

Reply:
The banner renders visible, and then was collapsed by the uswds JS being loaded. Depending on various factors that delay could be almost imperceptible to very noticeable. This version sets it to hidden immediately, as long as JS is running, while leaving it visible (as USWDS prefers) when JS doesn't run.

Reply:
Could we just remove the banner element, or does it do something important for the elements it contains?

Reply:
if we removed it there wouldn't be anything to read when you click the "Here's how you know" link.

Reply:
Oh no! I like reading that, so nm